interfaces/builtin/service_watchdog.go

```diff
@@ -53,15 +53,35 @@ func (iface *serviceWatchdogInterface) AppArmorConnectedPlug(spec *apparmor.Spec
 	notifySocket := osGetenv("NOTIFY_SOCKET")
```

Comment:
This is unrelated to this PR, but since you are touching this file, can you add a comment above this:
```
// If the system has defined it, use NOTIFY_SOCKET from the environment. Note
// this is safe because it is examined on snapd start and snaps cannot manipulate
// the environment of snapd. 
```

Reply:
Done